rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -3878,31 +3878,53 @@ void opcode_0160(const ScriptArguments& args, ScriptVec3 coord, const ScriptChan
     @arg arg3 Boolean true/false
     @arg blip Blip
 */
-void opcode_0161(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptBlipColour arg2, const ScriptBoolean arg3, ScriptBlip& blip) {
-    RW_UNIMPLEMENTED_OPCODE(0x0161);
-    RW_UNUSED(vehicle);
-    RW_UNUSED(arg2);
-    RW_UNUSED(arg3);
-    RW_UNUSED(blip);
-    RW_UNUSED(args);
+void opcode_0161(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptBlipColour colour, const ScriptBlipDisplay display, ScriptBlip& blip) {
+    auto& data = script::createObjectBlip(args, vehicle);
+    data.colour = colour;
+    /// @todo change ScriptBlipDisplay to BlipData::DisplayMode
+    data.display = static_cast <BlipData::DisplayMode> (display);
+    blip = &data;
 }
 
 /**
-    @brief %4d% = create_marker_above_actor %1d% color %2d% visibility %3d%
+    @brief ADD_BLIP_FOR_CHAR_OLD pedHandle colour display outblipHandle
 
-    opcode 0162
-    @arg character Character/ped
-    @arg arg2 
-    @arg arg3 
-    @arg blip Blip
+    opcode 0162 / 354
+
+    Creates and attaches blip for an ped
+
+    @arg pedHandle Ped
+    @arg colour Blip colour
+    @arg display Blip display mode
+    @arg outblipHandle Created blip
```

Comment:
The `@arg` was previously easily parsable by a computer. Now it's not. Check other instances and how we handled this in the past.

Reply:
I'm not sure I follow.
Was the name "argX" (where X is a number) easier to parse, or what is the actual difference?

Reply:
Take my commnt as an example on how *not* to write a review comment. I don't know what I meant myself.
But I *think* it has to do with the actual argument being called `blip` when this one refers to `outblipHandle`